chunky/src/res/se/llbit/chunky/ui/Credits.fxml

```diff
@@ -133,6 +133,16 @@
                                        <Insets left="20.0" />
                                     </padding>
                                  </Hyperlink>
+
+                                 <Hyperlink fx:id="openJdk" text="OpenJDK" />
+                                 <Hyperlink fx:id="openJdkLicense" text="GPLv2 + ce">
```

Comment:
Should we always display this or only in standalone mode (i.e. when launched using the standalone launcher)?